src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
I agree with @nicolas-grekas here. We don't have that many interfaces in Symfony that should almost *always* be implemented by the end user.